config/initializers/02_redis.rb

```diff
@@ -5,6 +5,8 @@
   ENV["REDIS_URL"] = redis_url if redis_url.present?
 end
 
-if ENV["REDIS_URL"].present? && Redis.current.ping != "PONG"
+REDIS = Redis.new(url: ENV["REDIS_URL"] || "")
```

Comment:
FYI on what's going on here. If the REDIS_URL is not present, we currently expect to raise a redis connection error. But weirdly, doing `Redis.new(url: nil)`will actually default to the localhost redis address and end up connecting successfully, so I'm using the or operator to pass an empty string, which makes redis throw the connection error we're expecting when REDIS_URL is not present.

Reply:
Additionally, this post recommends using the connection pool gem, happy to add this if we need, but I didn't want to go ahead an introduce another gem for the sake of it https://tejasbubane.github.io/posts/2020-04-22-redis-connection-pool-in-rails/

Reply:
Yep, lets go ahead and add that gem - I've read that it is strongly recommended